velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.h

```diff
@@ -17,8 +17,15 @@
 
 namespace facebook::velox::filesystems {
 struct HdfsServiceEndpoint {
-  std::string host;
-  int port;
+  HdfsServiceEndpoint(const std::string& hdfsHost, const std::string& hdfsPort)
+      : host(hdfsHost), port(hdfsPort) {}
+
+  std::string identity() const {
```

Comment:
add a comment above with an example of identity with and without port.

Reply:
Just added a comment for clarification. Thanks!